### PR TITLE
Rearranged the layer toggles and keys.

### PR DESCRIPTION
**Pull request by techtomas**
Moved the layer toggles to be in more comfortable locations for my typing. Also expanded the use of the media layer (now called APP) and enhanced the text navigation on the control layer

### CODE REVIEW

File: keyboard/ergodox_ez/keymaps/techtomas/readme.md [hunk 1]
```diff
@@ -6,28 +6,39 @@ The base layer is a merge of what I liked with the default layout and the ordina
 
 * The Caps Lock postion handles [Hyper/Esc](http://brettterpstra.com/2012/12/08/a-useful-caps-lock-key/)
 * The Tab key toggles MEH.
-* The two vertical mods on the left board are Shift+Tab and Tab.
-* The two vertical mods on the right board are Control Layer tap/toggle and Alt+Cmd.
+* The top vertical modes on the left and right are [ and ].
```
Comment (ezuk) [inline keyboard/ergodox_ez/keymaps/techtomas/readme.md]: Did you really mean to write `modes` here? Or was this supposed to say `mods`?

File: keyboard/ergodox_ez/keymaps/techtomas/readme.md [hunk 2]
```diff
@@ -6,28 +6,39 @@ The base layer is a merge of what I liked with the default layout and the ordina
...
 ## Symbol Layer
-The symbol layer can be tap/toggled through the bottom right key and also toggled holding the tilde/grave key in the upper left corner. I haven't used the symbol layer enough to tell if this was a good place or not but it made sense in my head
+The symbol layer can be tap/toggled with the bottom right key and also toggled on the thumb clusters.
 
 The bottom two vertical mods are set to F14 and F15 which is the default for changing screen brightness on the mac
 
-## Media Layer
-/ & ; will toggle the media layer which has play and volume controls assigned to the thumb cluster. I found this an easy way to quickly play/pause
+## App Layer
+The app layer is used to control media playback and also some application shortcuts.
```
Comment (ezuk) [inline keyboard/ergodox_ez/keymaps/techtomas/readme.md]: Would be cool to have a list of the application shortcuts. What do you think?

**Comment by ezuk** [comment]
Very nice @techtomas! Just a couple of comments.

File: keyboard/ergodox_ez/keymaps/techtomas/readme.md (hunk 1 above)
Comment (techtomas) [inline keyboard/ergodox_ez/keymaps/techtomas/readme.md]: Silly typo. Good catch I'll fix that.

File: keyboard/ergodox_ez/keymaps/techtomas/readme.md (hunk 2 above)
Comment (techtomas) [inline keyboard/ergodox_ez/keymaps/techtomas/readme.md]: I think that makes sense and is easy to do.

**Comment by techtomas** [comment]
Looking at this a 2nd time my application shortcuts are really more like convenience hotkeys. The intent was to take some of the repetitive key strokes I do every day and make them available to me in a number of different ways depending on were my fingers were on the keyboard. Would it be helpful to rewrite the description of the App layer to make it more clear what my vision for it was?

**Comment by ezuk** [comment]
IMHO the way you list the keys now makes this very clear. I'm going to merge this in - thank you for contributing! :) Please feel free to open new PRs as you polish your keymap in the future.

**Pull request merged**